ci-operator/step-registry/openshift-qe/installer/bm/prereqs/openshift-qe-installer-bm-prereqs-commands.sh

```
@@ -83,14 +99,19 @@ if [[ "$PRE_UEFI" == "true" ]]; then
   done
 fi
 EOF
-envsubst '${FOREMAN_OS},${LAB},${LAB_CLOUD},${NUM_WORKER_NODES},${PRE_PXE_LOADER},${PRE_UEFI},${QUADS_INSTANCE},${TYPE}' < /tmp/prereqs.sh > /tmp/prereqs-updated.sh
+envsubst '${FOREMAN_OS},${LAB},${LAB_CLOUD},${NUM_WORKER_NODES},${OCP_INVENTORY_OVERRIDE},${OCPINV},${PRE_PXE_LOADER},${PRE_UEFI},${QUADS_INSTANCE},${TYPE}' < /tmp/prereqs.sh > /tmp/prereqs-updated.sh
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)